misc/src/main/java/com/ihren/misc/config/ObjectMapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.ihren.misc.config;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class ObjectMapperConfig {
+    public static void configureObjectMapper(ObjectMapper objectMapper) {
```

Comment:
It is not necessary to have ObjectMapper suffix in method name.
You already have parameter as ObjectMapper